sdk/provisioning/Azure.Provisioning.CloudMachine/nuget.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
```

Comment:
Do we need this nuget.config? It looks to be the same as the root one.

Reply:
Oh, I didn't realize the root included the dev feed. Will remove!